packages/unpublished-eslint-rules/src/rules/prefer-static-type-import.js

```diff
@@ -0,0 +1,109 @@
+const collapseRanges = require('./utils/collapse-ranges');
```

Comment:
It's a bit odd that github is not doing syntax highlighting here, any ideas why?

Reply:
It is for me at least!  Also checked vscode